conan/api/output.py

```diff
@@ -108,12 +126,6 @@ def set_warnings_as_errors(cls, value):
 
     @classmethod
     def define_log_level(cls, v):
-        """
```

Comment:
Removed this as I don't think it should be callable by users and is an internal implementation of the verbosity levels